p2p/swarm.go

```diff
@@ -105,7 +107,7 @@ func (s *swarm) waitForGossip() error {
 func newSwarm(ctx context.Context, config config.Config, newNode bool, persist bool) (*swarm, error) {
 
 	port := config.TCPPort
-	address := inet.JoinHostPort("0.0.0.0", strconv.Itoa(port))
+	address := inet.JoinHostPort("127.0.0.1", strconv.Itoa(port))
```

Comment:
listening on 127.0.0.1 means you are only listening on loopback internal IF' listening on 0.0.0.0 means you are listening on any IF